src/AeroSharp/Connection/ClientWrapper.cs

```diff
@@ -16,5 +16,10 @@ public ClientWrapper(IAsyncClient client)
 
             Client = client;
         }
+
+        public Node[] GetClientNodes()
+        {
+            return Client.Nodes;
+        }
```

Comment:
As mentioned by Codecov, this code is untested. We should add a test that runs against the aerospike container and validate that it works as expected.

Reply:
Also, this could be more concisely expressed as an expression-bodied member: https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members

This would be, for example:
```suggestion
        public Node[] GetClientNodes() => Client.Nodes;
```

Reply:
that sounds good, I have made it property and not method - that's the only change i did. thanks much for your feedback.